api-participant/src/main/resources/api/openapi.yml

```diff
@@ -357,6 +357,7 @@ paths:
       parameters:
         - { name: portalShortcode, in: path, required: true, schema: { type: string } }
         - { name: envName, in: path, required: true, schema: { type: string } }
+        - { name: preferredLanguage, in: query, required: false, schema: { type: string } }
```

Comment:
This seems to me like something that belongs in the body, rather than the url, but it could go either way